virttest/qemu_vm.py

```diff
@@ -1205,6 +1205,9 @@ def add_boot(devices, boot_order, boot_once, boot_menu, boot_strict):
                 cmd += " %s" % ",".join(options)
             else:
                 cmd = ""
+            if 'reboottimeout' in params:
+                reboot_timeout = "reboot-timeout = %s" % params.get('reboottimeout')
```

Comment:
it should be `reboot_timeout = " reboot-timeout=%s" % params.get('reboottimeout')`

Reply:
it's better to provide a default value or use params["reboottimeout"]

Reply:
it's better to provide a special name, and use params.get("reboottimeout") instead of 'reboottimeout' in params

Reply:
@balamuruhans @suqinhuang  Thanks.  I updated this part code. 

            if params.get("seabios_reboottimeout"):
                reboot_timeout = " reboot-timeout = %s" % params.get('seabios_reboottimeout', 60)